flink-runtime/src/main/scala/org/apache/flink/runtime/akka/AkkaUtils.scala

```diff
@@ -505,12 +492,6 @@ object AkkaUtils {
          |      threshold = $transportThreshold
          |    }
          |
-         |    watch-failure-detector{
```

Comment:
Just to understand things, if we use these options here, why do they have no effect on akka?

Reply:
The `watch-failure-detector` configuration is relevant for Akka's deathwatch mechanism. Since we don't use this feature anymore, we can remove the configuration.